build/build_cross-platform.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+binary="portainer-$1-$2"
+
+docker run -tv $(pwd)/api:/src -e BUILD_GOOS="$1" -e BUILD_GOARCH="$2" portainer/golang-builder:cross-platform /src/cmd/portainer
+
+mkdir -p dist
+mv "api/cmd/portainer/$binary" dist/
+shasum "dist/$binary" > portainer-checksum.txt
```

Comment:
I'm wondering if we should remove the `shasum` step as it's not used at the moment (plus the binary is platform dependent, see the section about CentOS in http://portainer.readthedocs.io/en/stable/contribute.html).